src/locales/en/translation.json

```diff
@@ -145,5 +145,17 @@
     "prompt": "Please enter how many meals to gift",
     "totalLabel": "Total: ",
     "next": "Next"
+  },
+  "lyftReward": {
```

Comment:
I think you have to add this same (english) text the cn file--otherwise it'll come up as empty strings in the zh-cn UI

Reply:
ooh yeah good catch, updated the chinese!